### ISSUE TITLE
Some devices doesn't read the content of the Modals (i.e. Android Oreo , Voice Assistant)

### ISSUE THREAD

**Issue by vasildimitroffhristoff** [FundacionParaguaya/MentorApp#484]
(1) On some android devices the voice assistant does not read the accessibility label for modals (i.e. logout modal). 

(2) Add accessibility check for state of the elements - selected , deselected / enabled, disabled

**Comment by Danman7** [comment]
Closed in #548
